src/Presentation/GraphQL/Base/Query.cs

```diff
@@ -0,0 +1,13 @@
+using GraphQL.Types;
+
+namespace Presentation.GraphQL.Base
+{
+     public class Query : ObjectGraphType
+     {
+         public Query()
+         {
+             Name = "Query";
+             Field(typeof(StringGraphType), "BasicQuery",null, null, x => "Mutation!");
```

Comment:
Should we add a description instead of the first null field ?

Reply:
Also, I think that the resolver will return a "Query!" here :+1:

Reply:
Done